clusterDeployer.py

```diff
@@ -468,21 +481,30 @@ def deploy(self) -> None:
             else:
                 logger.info("Skipping pre configuration.")
 
-            lh = host.LocalHost()
-            self.ensure_linked_to_bridge(lh)
+            if self._cc.kind != "microshift":
+                lh = host.LocalHost()
+                self.ensure_linked_to_bridge(lh)
 
-            if "masters" in self.steps:
-                self.teardown()
-                self.create_cluster()
-                self.create_masters()
-            else:
-                logger.info("Skipping master creation.")
-
-            if "workers" in self.steps:
-                if len(self._cc.workers) != 0:
-                    self.create_workers()
+                if "masters" in self.steps:
+                    self.teardown()
+                    self.create_cluster()
+                    self.create_masters()
                 else:
-                    logger.info("Skipping worker creation.")
+                    logger.info("Skipping master creation.")
+
+                if "workers" in self.steps:
+                    if len(self._cc.workers) != 0:
+                        self.create_workers()
+                    else:
+                        logger.info("Skipping worker creation.")
+        if self._cc.kind == "microshift":
+            version = match_regex = self._cc.version
```

Comment:
i think you meant to do
`version = match_to_proper_version_format(self._cc.version)`?

I don't see match_regex used anywhere.

Reply:
thanks!!